rules/Proj0501.md

```diff
@@ -0,0 +1,41 @@
+---
+parent: Licening
+ancestor: Rules
+---
+
+# Proj0501: Only include packages with a compliant license
+Using a [NuGet](https://www.nuget.org) (third-party) package implies that you
+and/or your company explicitly agree with the legally binding conditions of the
+license and the copyright of the onwer of the package.
+
+By default, MIT and Apache-2.0 are allowed.
```

Comment:
Why not the whole set of permissive (and non-viral copyleft) licenses?

Reply:
That is even better. This was what I was thinking of when I was typing.

Reply:
And I think we have to make that list explicit in our documentation.

Reply:
I would be in favor of leaving this docs out of this PR for now, since it is heavily related to [dotnet-project-file-analyzers#328](https://github.com/dotnet-project-file-analyzers/dotnet-project-file-analyzers/pull/328)

Reply:
I think we can adjust is afterwards too. And it is something we are also already working on!?